actix-web/Cargo.toml

```diff
@@ -72,6 +72,9 @@ rustls-0_21 = ["http2", "actix-http/rustls-0_21", "actix-tls/accept", "actix-tls
 # TLS via Rustls v0.22
 rustls-0_22 = ["http2", "actix-http/rustls-0_22", "actix-tls/accept", "actix-tls/rustls-0_22"]
 
+# Full unicode support
+unicode = ["dep:regex", "actix-router/unicode"]
```

Comment:
It seems that this broke repos that are using actix-web from git? The `unicode` feature is not available in actix-router 0.5, but it is required here:

```
error: failed to select a version for `actix-router`.
    ... required by package `actix-web v4.5.1 (workspace/actix-web/actix-web)`
    ... which satisfies path dependency `actix-web` of package `xxx v0.x.x (xxx)`
versions that meet the requirements `^0.5` are: 0.5.2, 0.5.1, 0.5.0

the package `actix-web` depends on `actix-router`, with features: `unicode` but `actix-router` does not have these features.



failed to select a version for `actix-router` which could resolve this conflict
```

Reply:
@SteveLauC

```toml
[dependencies]
actix-web = { git = "https://github.com/actix/actix-web.git", rev = "c10f05a8672e53ecada8ec2f15bf9d927efade57", default-features = false, features = [
    "macros",
] }
[patch.crates-io]
actix-router = { git = "https://github.com/actix/actix-web.git", rev = "c10f05a8672e53ecada8ec2f15bf9d927efade57" }
```

Reply:
Yeah, that would make things work!